modules/kmm-adding-the-keys-for-secureboot.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-adding-the-keys-for-secureboot_{context}"]
+= Adding the keys for secureboot
+
+To use KMM Kernel Module Management (KMM) to sign kernel modules a certificate and private key are required.
+For details on how to create these, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_monitoring_and_updating_the_kernel/signing-kernel-modules-for-secure-boot_managing-monitoring-and-updating-the-kernel#generating-a-public-and-private-key-pair_signing-kernel-modules-for-secure-boot[Generating a public and private key pair].
+
+.Procedure
+
+. Create the `my_signing_key_pub.der` file that contains the certificate and the `my_signing_key.priv` file that contains the private key:
++
+[source,terminal]
+----
+openssl req -x509 -new -nodes -utf8 -sha256 -days 36500 -batch -config configuration_file.config -outform DER -out my_signing_key_pub.der -keyout my_signing_key.priv
```

Comment:
This line is now different in the rhel docs because they've added wrappers around the key-pair generation so we should probably remove it from the openshift docs, it will work to create them but we probably want to be consistent with the official RHEL method. 
It should stay in the upstream docs though because its correct.